packages/fx-core/resource/package.nls.json

```diff
@@ -1,79 +1,79 @@
-  "core.provision.confirmSubscription": "You are switching away from your previous subscription %s.\nDo you want to provision resources in %s environment? Azure account (%s) with subscription (%s) will be used to provision the resources. Cost may incur according to the usage.",
+  "core.provision.confirmNotice": "Teams Toolkit will provision resources and register corresponding resource providers under Azure account (%s) with subscription (%s). Costs may incur according to the pricing calculator.",
+  "core.provision.confirmEnvNotice": "Do you want to provision resources for the '%s' environment?\nAzure account (%s) with subscription (%s) will be used to provision the resources. Costs may incur according to the usage.",
+  "core.provision.confirmSubscription": "You are switching away from your previous subscription %s.\nDo you want to provision resources for the '%s' environment? Azure account (%s) with subscription (%s) will be used to provision the resources. Cost may incur according to the usage.",
   "core.provision.provision": "Provision",
   "core.provision.learnMore": "Learn more",
   "core.provision.switchedM365AccountNotice": "You are now using a different Microsoft 365 tenant from what you previously used.",
   "core.provision.switchedAzureSubscriptionNotice": "You are now using a different Azure subscription from what you previously used.",
-  "core.provision.switchedM365AccountAndAzureSubscriptionNotice": "You are now using a different Microsoft 365 tenant and a differnt Azure subscription from what you previously used.",
-  "core.provision.createNewAzureBotNotice": "We will create a new Azure bot resource.",
+  "core.provision.switchedM365AccountAndAzureSubscriptionNotice": "You are now using a different Microsoft 365 tenant and a different Azure subscription from what you previously used.",
+  "core.provision.createNewAzureBotNotice": "Teams Toolkit will create a new Azure bot resource.",
   "core.provision.azureAccount": "Azure account: %s",
   "core.provision.azureSubscription": "Azure subscription: %s",
   "core.provision.m365Account": "Microsoft 365 account: %s",
-  "core.provision.confirmEnvAndCostNotice": "Cost may incur according to the usage. Do you want to provision resources in %s environment using accounts listed above?",
-  "core.provision.confirmEnvOnlyNotice": "Do you want to provision resources in %s environment using the account listed above?",
-  "core.deploy.confirmEnvNotice": "Do you want to deploy resources in %s environment?\nAzure account (%s) with subscription (%s) will be used for deployment.",
-  "core.deploy.confirmEnvNoticeV3": "Do you want to deploy resources in %s environment?",
+  "core.provision.confirmEnvAndCostNotice": "Costs may incur according to the usage. Do you want to provision resources for the '%s' environment using the accounts listed above?",
```

Comment:
accounts -> account

If this refers to Azure account, at any given time, only one active account is allowed.